### ISSUE TITLE
Binded gamepad button for autoload menu in the shop doesn't work

### ISSUE THREAD

**Issue by goomba00** [loki79uk/FS25_UniversalAutoload#188]
If any gamepad button binded for autoload menu in the shop, it doesn't work (but it shows correctly in the lower part of the screen).

**Comment by loki79uk** [comment]
Okay, thanks - I'll see if I can work out an alternative method for the keybinding.  I might have to actually buy a controller for testing.. :D

**Comment by loki79uk** [comment]
Can you try this again with the latest version please: https://github.com/loki79uk/FS25_UniversalAutoload/releases

**Comment by goomba00** [comment]
v0.3.2.8 - still not working

**Comment by loki79uk** [comment]
Which button are you trying to use?  It has to be different to the other menu buttons.

**Comment by goomba00** [comment]
I'm using L-stick press. Tried other buttons too.

**Comment by geeknessfr** [comment]
Tried with LB+X, shortcut not showing completely but working to open the menu :

![image](https://github.com/user-attachments/assets/b244b621-c2e3-4e8e-bcce-ed87a3d6b77f)
![image](https://github.com/user-attachments/assets/ea2ea359-c9ac-42f3-9188-5c8d791cdb79)

But simple buttons like, A, B, X, Y or LB / RB are used by default in the shop, so they can't be used as a shortcut.

Tested with L-Stick/R-Stick and they don't work either, I think the game keep them as a default shortcut for anything else...

**Comment by goomba00** [comment]
> Tried with LB+X, shortcut not showing completely but works fine to open the menu :
> 
> **But simple buttons like, A, B, X, Y or LB / RB are used by default in the shop, so they can't be used as a shortcut.**
> 
> Tested with L-Stick/R-Stick and they don't work either, I think the game keep them as a default shortcut for anything else...

This not exactly works. Because RB & LB are used to scroll to right-left for options in the left menu (if scrolling availble for them). So it works only if cursor set on the option whithout scrolling possibility (like color or sign choosing). In other options (first ones are usually scrollable, like tyres selection or configuration selection) when you just press RB scrolling begins and you have to press RB+X very quickly, but anyway scrolling will happen and your tyres will be changed before you enter to Autoload menu.

**Comment by loki79uk** [comment]
The point is you can't use buttons that are already bound.  There is nothing I can do about it. Either you need to unbind the controller buttons you do not use, or use one of the keys on the keyboard.  This is how the GIANTS key binding system works, it is not something I have any control over.

**Comment by goomba00** [comment]
Just tried LS+X combination and it works good.

**Comment by goomba00** [comment]
> The point is you can't use buttons that are already bound. There is nothing I can do about it. Either you need to unbind the controller buttons you do not use, or use one of the keys on the keyboard. This is how the GIANTS key binding system works, it is not something I have any control over.

In fact in other cases it works. You can use combinations if they are already binded somethere. If they not used in same situation  (like autoload and trees harvesting) it works fine.

**Comment by loki79uk** [comment]
Yes of course, it is the same for every action in the game.  It should have been obvious that I was talking about actions in the shop menu.

**Comment by loki79uk** [comment]
Closing as not a bug.  Please open a ticket with GIANTS if you can suggest a better way for their key bindings system to work.

**Comment by goomba00** [comment]
I don't think that any more fixes are needed. 
Combinations with **LS** are working fine. **LS+X** for example. Thanks for support!
May be it just will be useful to add some comments about it in the mod description.

**Comment by goomba00** [comment]
Hi. Just installed Financing mod from Modhub and is uses L-Stick button in shop menu. Looks like it could be possible to bind it somehow.
https://www.farming-simulator.com/mod.php?mod_id=308556&title=fs2025

**Comment by loki79uk** [comment]
> Hi. Just installed Financing mod from Modhub and is uses L-Stick button in shop menu. Looks like it could be possible to bind it somehow. https://www.farming-simulator.com/mod.php?mod_id=308556&title=fs2025

I guess anything is possible if the key is hard coded, but not everyone wants to use the same keys.  Better to make as specific action with its own user-definable key in the option menu.  Also, if another mod does it the same way for the same key in the same menu then you will get an unresolvable conflict.
